sources/engine/Stride.Physics/Elements/CollisionFilterGroups.cs

```diff
@@ -79,4 +79,31 @@ public enum CollisionFilterGroupFlags
 
         AllFilter = 0xFFFF,
     }
+
+    /// <summary>
+    /// Flags that control how ray tests are performed
+    /// </summary>
+    [Flags]
+    public enum EFlags : uint
```

Comment:
Is the name EFlags appearing in Bullet? If not, can we rename it to RayCastFlags or something?

Reply:
Yes, see the source over here https://github.com/bulletphysics/bullet3/blob/39b8de74df93721add193e5b3d9ebee579faebf8/src/BulletCollision/NarrowPhaseCollision/btRaycastCallback.h#L32
I would have also preferred to change the name, but it would be far easier for our users to find resources around this if we keep the same name. What do you think ?

Reply:
Agreed - we want to keep Bullet naming where possible so users can use Bullet forums and stuff with Stride.